Samples/iOS-Swift/iOS-SwiftUITests/UIEventBreadcrumbTests.swift

```diff
@@ -22,17 +22,17 @@ class UIEventBreadcrumbTests: XCTestCase {
         app.buttons["Extra"].tap()
         app.buttons["UI event tests"].tap()
 
-        let label = app.staticTexts["breadcrumbLabel"]
-
         //Trigger a change in textfield
         app.buttons["editingChangedButton"].tap()
+
+        Thread.sleep(forTimeInterval: 0.5)
```

Comment:
`m`: Consider using `waitForExistence` with a timeout instead, please.

Reply:
the problem is that there is nothing to "waitForExistence"

Reply:
I'll rewrite the sample to dynamically create labels for every step. Overkill but it will solve

Reply:
So, I was using `accessibilityIdentifier` and totally forgot that it is possible to wait for a label with a specific text.
Now I have a proper solution.

Reply:
So, I was so fixed to use `accessibilityIdentifier` that I forgot that is possible to wait for a label with a specific text.
Now it works, but I had to update the macOS and Xcode version.